src/main/frontend/worker/db_worker.cljs

```diff
@@ -197,21 +246,27 @@
       (.exec db "PRAGMA locking_mode=exclusive")
       (sqlite-common-db/create-kvs-table! db)
       (sqlite-common-db/create-kvs-table! client-ops-db)
+      (db-migrate/migrate-sqlite-db db)
```

Comment:
`migrate-sqlite-db` must be run before `db-migrate/migrate` because DataScript needs to be restored from the underlying SQLite db.